config/sync/core.entity_view_display.user.user.default.yml

```diff
@@ -16,21 +16,13 @@ bundle: user
 mode: default
 content:
   member_for:
-    weight: 1
-    settings: {  }
-    third_party_settings: {  }
+    weight: 5
```

Comment:
This appears to be reverting the change I made to hide the profile types from the user view mode.